client/src/components/PostBody/PostBody.js

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import "./PostBody.css";
+import "../../App.css";
```

Comment:
You shouldn't need to import the `App.css` file anymore - it's been added to the storybook global config

Reply:
Removed this. Bill's going to import the bold version of open sans.